Meter.h

```diff
@@ -134,6 +134,13 @@ typedef enum {
    LAST_METERMODE
 } MeterModeId;
 
+typedef enum {
+   DATA_RATESTATUS,
+   INIT_RATESTATUS,
+   NODATA_RATESTATUS,
+   STALE_RATESTATUS
```

Comment:
I'd prefer to have the namespacing of those constants to be prefixed. This also aligns better with most of the remaining code (Meter.h being an exception here AFAICS.

```suggestion
   RATESTATUS_DATA,
   RATESTATUS_INIT,
   RATESTATUS_NODATA,
   RATESTATUS_STALE
```